tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2841,6 +3101,81 @@ <h2>Create a distance feature</h2>
         this.updateCorrectness_();
       },
 
+      sortFeatureValues: function(values) {
+        const inferenceStats = this.inferenceStats_[0];
+        values = values.filter(a => {
+          if (this.isRegression_(this.modelType)) {
+            return this.getRegressionCount(inferenceStats, a) > 0
+          } else if (this.isBinaryClassification_(
+            this.modelType, this.multiClass)) {
+            return this.getBinaryClassificationCount(inferenceStats, a) > 0
+          } else {
+            return this.getMulticlassClassificationCount(
+              inferenceStats, a) > 0;
+          }
+        });
+        return values.sort((a, b) => {
+          if (this.selectedFeatureSort == 'Count') {
+            if (this.isRegression_(this.modelType)) {
+              return this.getRegressionCount(inferenceStats, b) -
+                  this.getRegressionCount(inferenceStats, a);
+            } else if (this.isBinaryClassification_(
+              this.modelType, this.multiClass)) {
+              return this.getBinaryClassificationCount(inferenceStats, b) -
+                  this.getBinaryClassificationCount(inferenceStats, a);
+            } else {
+              return this.getMulticlassClassificationCount(
+                inferenceStats, b) - this.getMulticlassClassificationCount(
+                  inferenceStats, a);
+            }
+          } else if (this.selectedFeatureSort == 'Alphabetical') {
+            return this.getPrintableValue_(a).localeCompare(this.getPrintableValue_(b));
+          } else if (this.selectedFeatureSort == 'Accuracy') {
+            console.log('a acc' + this.getAccuracyModelIndex(
```

Comment:
If not intentional I would remove console.log.

Reply:
removed